starwars/src/components/StarWarsData.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './StarWars.css';
+
+const Chars = props => {
```

Comment:
remember here also to correct all state to lowercase. after I applied all the changes I have mentioned your code works.